tests/src/Type/EntityQueryDynamicReturnTypeExtensionTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace mglaman\PHPStanDrupal\Tests\Type;
```

Comment:
Shouldn't the test be in a consistent namespace with `mglaman\PHPStanDrupal\Type\EntityQuery\EntityQueryDynamicReturnTypeExtension`?

Something like `mglaman\PHPStanDrupal\Tests\Type\EntityQuery`?

Reply:
Yeah we can move it there

Reply:
Done!